english/jacket/milk-and-cookies/jacket-cookie.md

```diff
@@ -0,0 +1 @@
+As you take the last bite of a 32 pack of Oreo's you realize the milk was actually perfectly liquefied marshmallows.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.